clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -2430,6 +2430,9 @@ llvm::DISubprogram *CGDebugInfo::CreateCXXMemberFunction(
 
   SPCache[Method->getCanonicalDecl()].reset(SP);
 
+  // Add the method declaration as a call target.
+  addCallTarget(MethodLinkageName, SP, /*CI=*/nullptr);
```

Comment:
Possibly dumb question: What is this call doing?

Reply:
There are 3 items that are required to have a complete entry for the call-site information:
``Function Name``, ``Metadata Node`` and ``Instruction``.
They are collected as a whole set or individually. Here we have only ``Function Name``, ``Metadata Node``. The ``Instruction`` was collected or will be collected, in order to have the full set.